include/sdf/config.hh.in

```diff
@@ -11,18 +11,18 @@
 #define SDF_VERSION "${SDF_PROTOCOL_VERSION}"
```

Comment:
I think we should add header guards to this file and change the comment on line 1 from `config.h` to `config.hh`

Reply:
This is copied from https://github.com/ignitionrobotics/sdformat/blob/main/cmake/sdf_config.h.in which does not have include headers as well thinking . Maybe it's because it only contains #define so it doesn't matter if it is included multiple times?

Reply:
I see that we don't have header guards in many of the `config.hh.in` files, but I think that is an oversight. To be safe I would just add header guards